packages/http/fetch/src/middlewares/retryHandler.ts

```diff
@@ -165,14 +165,18 @@ export class RetryHandler implements Middleware {
 		if (retryAttempts < currentOptions.maxRetries && this.isRetry(response) && this.isBuffered(fetchRequestInit) && currentOptions.shouldRetry(currentOptions.delay, retryAttempts, url, fetchRequestInit as RequestInit, response)) {
 			++retryAttempts;
 			setRequestHeader(fetchRequestInit, RetryHandler.RETRY_ATTEMPT_HEADER, retryAttempts.toString());
+      let delay = null;
 			if (response) {
-				const delay = this.getDelay(response, retryAttempts, currentOptions.delay);
+        delay = this.getDelay(response, retryAttempts, currentOptions.delay);
 				await this.sleep(delay);
 			}
 			if (tracerName) {
 				return await trace.getTracer(tracerName).startActiveSpan(`retryHandler - attempt ${retryAttempts}`, (span) => {
 					try {
-						span.setAttribute("http.retry_count", retryAttempts);
+            span.setAttribute("http.request.resend_count", retryAttempts);
+            if(delay){
+              span.setAttribute("http.request.resend_delay", delay);
+            }
 						span.setAttribute("http.status_code", response.status);
```

Comment:
```suggestion
						span.setAttribute("http.response.status_code", response.status);
```

(there are probably other places)